src/Support/Arr.php

```diff
@@ -49,6 +49,24 @@ public static function arrayMergeRecursiveDistinct(array &$array1, array &$array
         return $merged;
     }
 
+    /**
+     * @param $array1
+     * @param $array2
+     *
+     * @return array
+     */
+    public static function mergeAfterEachOther($array1, $array2): array
```

Comment:
strict typing toepassen. Zie ook phpdocs

Reply:
done
array1 = object en is pas supported vanaf php 7.2
array2 = array -> toegevoegd